AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -129,8 +137,20 @@ - (BOOL)saveLog:(id<MSLog>)log withGroupId:(NSString *)groupId flags:(MSFlags)fl
 
 - (NSUInteger)countLogsBeforeDate:(NSDate *)date {
   long long timestampMs = (long long)([date timeIntervalSince1970] * 1000);
-  NSMutableString *condition = [NSMutableString stringWithFormat:@"\"%@\" <= '%lld'", kMSTimestampColumnName, timestampMs];
-  return [self countEntriesForTable:kMSLogTableName condition:condition];
+  NSMutableString *condition = [NSMutableString stringWithFormat:@"\"%@\" <= ?", kMSTimestampColumnName];
+  return [self countEntriesForTable:kMSLogTableName condition:condition withValues:@[[NSNumber numberWithLongLong:timestampMs]]];
+}
+
+- (NSString *)buildKeyFormatWithCount:(unsigned long)count {
```

Comment:
`componentsJoinedByString`?

Reply:
We will still have to construct the array